soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -163,9 +166,12 @@ __asm__(".align 4\n\t"
 	"  wsr   a1, WINDOWSTART\n\t"
 	"  wsr   a0, WINDOWBASE\n\t"
 	"  rsync\n\t"
-	"  movi  sp, d0i3_stack\n\t"
-	"  movi a2, 0x1000\n\t"
-	"  add sp, sp, a2\n\t"
+	"  movi  sp, z_interrupt_stacks\n\t"
+	"  rsr   a2, PRID\n\t"
+	"  movi  a3, 0x800\n\t"
```

Comment:
Could you try `"movi a3, " CONFIG_ISR_STACK_SIZE "\n\t"`? This should embed the value and you can skip the BUILD_ASSERT above.

Reply:
`"movi a3, " STRINGIFY(CONFIG_ISR_STACK_SIZE)  "\n\t"` should work ;)